### PR TITLE
Fix universal linking not working on android 12.

### PR DESCRIPTION
**Pull request by matthewcarlreetz**

### CODE REVIEW

File: android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -31,14 +31,14 @@
                 <category android:name="android.intent.category.LAUNCHER" />
 
             </intent-filter>
-            <intent-filter>
+            <intent-filter android:autoVerify="true">
```
Comment (matthewcarlreetz) [inline android/app/src/main/AndroidManifest.xml:34]: This flag is required on to verify on android 12

File: android/app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -31,14 +31,14 @@
...
                 <action android:name="android.intent.action.VIEW" />
                 <category android:name="android.intent.category.DEFAULT" />
                 <category android:name="android.intent.category.BROWSABLE" />
-                <data android:scheme="helium" />
```
Comment (tyler-whitman) [inline android/app/src/main/AndroidManifest.xml:38]: We dont need this scheme anymore?

**Review by tyler-whitman — APPROVED** [review]

File: android/app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (matthewcarlreetz) [reply android/app/src/main/AndroidManifest.xml:38]: We do, but it's in a different intent-filter

**Comment by ChunChiehKung** [comment]
I still has the issue.
Is there any change I need to do in maker start app?
Please advise.
Thanks a lot.

**Pull request merged**